### ISSUE TITLE
NTR: Anti FR1 antibodies (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8748]
New term request Anti FR1 antibodies (segal_110906202852): Folate receptor-alpha

**Comment by pnrobinson** [comment]
@MickeySegal
What is the context? Is this anti-heavy chain? Would we need to specify Ig type?

**Comment by MickeySegal** [comment]
https://pubmed.ncbi.nlm.nih.gov/20668945/
"Generation of autoantibodies against the folate receptor required to transport 5MTHF into CSF and mutations in the folate receptor 1 (FOLR1) gene have been reported to be causes of CFD"

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000795
- label: Anti-folate receptor antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:38068740
- def: The presence of autoantibodies (immunoglobulins) in the serum that react against the folate receptor in the blood circulation.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8748
- synonyms:
